Parsers/ASimRegistryEvent/Parsers/VimRegistryEventMicrosoftSecurityEvent.yaml

```diff
@@ -0,0 +1,284 @@
+Parser:
```

Comment:
please rename this file to 'vimRegistryEventMicrosoftSecurityEvent.yaml'

Reply:
done

Reply:
@Alekhya0824 please check, this is not reflecting

Reply:
done

Reply:
The file was copied to ProcessEvent folder, I moved it back.